src/generated/api/.openapi-generator/FILES

```diff
@@ -1,5 +1,6 @@
 .gitignore
 .npmignore
+.openapi-generator-ignore
```

Comment:
Is this file something that's generated? If it was a "rename" manually, no need to store it here.

Reply:
Note that the path is `src/generated/api/...`. Yes, this is a generated file